src/footer/footergrid.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+
+import { useStyles } from 'footer/styles'
+
+export const Footergrid = () => <Grid>
+  <p className={useStyles().paragraph}>
+    Made with love by
+    <a href={'about.jsx'}> Hermes team</a>
```

Comment:
We should remove this `.jsx` extension from the `<a/>` tag, as the link to the About page will not contain this extension.

Reply:
Done